datamade/committee-oversight
# Committee Detail pages

## Overview

Closes #66. The design of the committee detail pages—as far as we can get before introducing the data. Here's what it should look like: 

![screencapture-localhost-8000-committee-46d6937b-9758-4e9f-80ef-7e270c84e57e-2019-10-28-14_48_01](https://user-images.githubusercontent.com/1094243/67712461-040ccd00-f992-11e9-804d-8fd2c8a55748.png)

## Testing Instructions

 * If you don't already have it, set up/reset the site with the following steps:
	* Delete your volume (may not be necessary): `docker-compose down --volumes`
	* Start the site back up: `docker-compose up`
	* Open a new window and load `hearings.dump`:  `docker-compose exec postgres pg_restore -C -j4 --no-owner -U postgres -d hearings /app/hearings.dump`
	* Run migrations: `docker-compose exec app python manage.py migrate`
	* Load CMS content: `docker-compose run --rm app python manage.py load_cms_content`
	* Load sample committee ratings: `docker-compose run --rm app python manage.py load_committeeratings`
* Start up the site: `docker-compose up`
* Create a new committee page with a chairperson's name
* View it live at various sizes. Does it look good + like what you'd expect?
